Services/UserService/Commands/DeleteUserCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿ using System;
+ using System.Net;
+ using IDeleteUserUserService.Interfaces;
+
+
+ namespace UserService.Commands
+
+ {
+     public class DeleteUserCommand : IDeleteUser
+     {
+         public HttpStatusCode Execute(int userId)
+         {
+             return DeleteUserFromDataBaseService(userId) == "Ok" ? HttpStatusCode.OK : HttpStatusCode.BadRequest;
+         }
+
+        //TODO fix from stub on request
+         private string DeleteUserFromDataBaseService(int userId)
```

Comment:
Not sure about **FromDataBaseService**, imho it is not good name and why this method return something?

Reply:
> Not sure about **FromDataBaseService**, imho it is not good name and why this method return something?

it is the stub